tool/presubmit.sh

```diff
@@ -3,19 +3,47 @@
 # Fast fail the script on failures.
 set -e
 
-pushd bazel_codegen
-pub upgrade
-dartanalyzer --fatal-warnings lib/_bazel_codegen.dart
-pub run test
-popd
+e2e_tests () {
+  if [ "$TRAVIS" == "true" ]; then
+    echo "Installing bazel"
+    # Install bazel if we are on travis.
+    wget https://github.com/bazelbuild/bazel/releases/download/0.4.4/bazel_0.4.4-linux-x86_64.deb
+    sudo dpkg -i bazel_0.4.4-linux-x86_64.deb
```

Comment:
This doesn't work with Travis containers. I think we can add some config to get around it but I'm not sure it's a good idea.

Was the goal to get faster feedback from unit tests?